package.json

```diff
@@ -69,6 +69,7 @@
     "jest": "^26.1.0",
     "jest-environment-jsdom-sixteen": "^1.0.3",
     "jest-extended": "^0.11.5",
+    "jest-when": "^2.7.2",
```

Comment:
Nice find! It's good that there's a jest library that closely resembles sinon's `withArgs` API